feature-detects/dom/shadowroot.js

```diff
@@ -0,0 +1,22 @@
+/*!
+{
+  "name": "Shadow DOM API",
+  "property": ["shadowroot", "shadowrootlegacy"],
```

Comment:
please split up again in two files, that way we can get rid of one if we want to deprecate the old one